src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -176,6 +176,17 @@ default String embedFooterIcon() {
         return "https://github.com/pajlads/DinkPlugin/raw/master/icon.png";
     }
 
+    @ConfigItem(
+        keyName = "rsnDenyList",
+        name = "RSN Deny List",
+        description = "Prevent notifications from the following comma-separated player names",
```

Comment:
For webhooks, we only split on new line. I think following that scheme (at least for how we document/explain the setting) is beneficial

Reply:
Wording?
```suggestion
        description = "Prevent notifications from the following player names (on separate lines)",
```

Reply:
@Felanbird

Reply:
`Prevent notifications from the following player names (One name per line)`